src/docs/markdown/caddyfile/options.md

```diff
@@ -400,6 +400,33 @@ http:// {
 }
 ```
 
+<aside class="tip">
+
+If you are defining the [`bind` directive](/docs/caddyfile/directives/bind) inside a site or the [`default_bind` global option](/docs/caddyfile/options#default_bind), the `servers [<listener_address>]` block *MUST* match the address defined on either directives, otherwise the settings won't be applied.
+
+</aside>
+	
+For example:
+
+```caddy
+{
+	servers :8080 {  <--- THIS WILL NOT MATCH ANY LISTENER
+		name private
+	}
+	
+	servers 192.168.1.2:8080 {  <--- THIS WILL WORK BECAUSE IS AN EXACT MATCH 
```

Comment:
Let's use Caddyfile comments so that it gets highlighted correctly. Also we don't need to use caps, it's too aggressive to read in docs.

```suggestion
	# This will NOT match any listener
	servers :8080 {
		name private
	}
	
	# This will work because it's an exact match
	servers 192.168.1.2:8080 { 
```

Reply:
Thanks, makes total sense!